roles/update/tasks/create-instance.yml

```diff
@@ -0,0 +1,20 @@
+- name: Create an instance on the overcloud
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    PATH: "{{ cifmw_path }}"
+  ansible.builtin.shell: |
+    set -o pipefail
+    cat {{ cifmw_update_workload_launch_script }} | \
+       oc rsh -n {{ cifmw_install_yamls_defaults['NAMESPACE'] }} openstackclient bash 2>&1 \
+       {{ cifmw_update_timestamper_cmd }} | tee {{ cifmw_update_artifacts_basedir }}/workload_launch.log
+  when: not cifmw_update_run_dryrun | bool
```

Comment:
You have the same conditional in main.yaml block, so here it is not required

Reply:
Done